src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -98,11 +96,9 @@ public function formatBatch(array $records): mixed
         return $records;
     }
 
-    private function doFormat(array|LogRecord $record): mixed
+    public function format(LogRecord $record): mixed
     {
-        if ($record instanceof LogRecord) {
-            $record = $record->toArray();
-        }
+        $record = $record->toArray();
```

Comment:
Strictly speaking here you don't need to do this anymore I think, you could simply work with $record values below, this was done for monolog2 compat.

Reply:
Fixed